spec/set.js

```diff
@@ -0,0 +1,12 @@
+// import chai, { expect } from 'chai'
```

Comment:
Please do not submit commented out code - one option is to stash this file, to avoid creating noise in the PR that distracts from the features under review.